app/assets/javascripts/objects_input.js

```diff
@@ -1,61 +1,54 @@
-function loadObjectInputs(elem) {
```

Comment:
This function is still used in various places to enable the tagging for elements that are added to the page dynamically

https://github.com/seek4science/seek/blob/902167afe1405cbad6bc8cdee7553bfc504840de/app/views/data_files/create_content_blob.js.erb#L30